app/models/folio/newsletter_subscription.rb

```diff
@@ -83,10 +77,6 @@ def update_mailchimp_subscription(email_for_subscription)
         Folio::Mailchimp::CreateOrUpdateSubscriptionJob.perform_later(email_for_subscription)
       end
     end
```

Comment:
Tady to možná chtělo předělat na  kontrolu  že `self.site == user.auth_site`. Nic klíčového, ale zabrání to  nečekaným potížím (které ale nemusí nikdy nastat).